src/utp_socket_manager.cpp

```diff
@@ -161,7 +161,7 @@ namespace libtorrent
 	{
 //		UTP_LOGV("incoming packet size:%d\n", size);
 
-		if (p.size() < int(sizeof(utp_header))) return false;
+		if (p.size() < sizeof(utp_header)) return false;
```

Comment:
I don't understand why this does not triggered the warning `sign-compare` before, hopefully with the new typed indexes will be more easy to enable `sign-conversion`